### PR TITLE
Updating bug bounty pages with changes to program

### PR DESCRIPTION
**Pull request by rforbes**
I have added a couple new pages and expanded the pages to the current bug bounty program. This includes new payouts, links to the hall of fame, and clarification on requirements.

### CODE REVIEW

File: vendor-local/src/legal-docs [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit f58ed34946907d108f44f829167de904d0675324
+Subproject commit 21758916986f033b4368f6311b0acbc2bdbeb0dd
```
Comment (jgmize) [inline vendor-local/src/legal-docs]: This looks like it was unintentionally included in this PR.

Comment (rforbes) [inline vendor-local/src/legal-docs]: you might be right.  I know I did not intentionally change any files in that directory.  I am not actually sure what to do about it as this isn't a specific file.  I am not really familiar with subprojects.

Comment (jgmize) [inline vendor-local/src/legal-docs]: No worries, we'll fix this on merge.

File: bedrock/security/templates/security/bug-bounty.html [hunk 2]
```diff
@@ -19,112 +19,14 @@ <h1 itemprop="name" class="title-shadow-box">Bug Bounty Program</h1>
...
-        <li>Security bug must be a remote exploit.</li>
-        <li>Submitter must not be the author of the buggy code nor otherwise involved
-          in its
-          contribution to the Mozilla project (such as by providing check-in
-          reviews).
-        </li>
-        <li>Employees of the Mozilla Foundation and its subsidiaries are
-          ineligible.
-        </li>
+        <li>Information on the Client Bug Bounty Program can be found <a href="{{ url('security.client-bug-bounty') }}">Here</a>
```
Comment (jpetto) [inline bedrock/security/templates/security/bug-bounty.html]: Small nit - make "Here" lowercase.

File: bedrock/security/templates/security/client-bug-bounty.html [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+      <p>If two or more people report the bug together the
+        reward will be divided among them. </p>
+
+      <h2>Reward Guidelines</h2>
+
+      <p>Mozilla will pay a bounty for client and security bugs as detailed below. All security bugs must follow the following general criteria to be eligible:</p>
+
+       <p> Eligible security bugs may be present in any of the current main development or released versions of Firefox, Thunderbird, Firefox for Android, or FirefoxOS as released by Mozilla Corporation (e.g. Nightly mozilla-central, Firefox Developer Edition, or Beta test versions, as well as the final release product versions)</p>
+
+        <p>The security rating given by Bounty Committee for a bug must rated a "sec-critical" or a "sec-high" in order for it to be eligible for a bounty. Some "sec-moderate" bugs may be eligible for the bounty as well. (See https://wiki.mozilla.org/Security_Severity_Ratings for details of the rating qualifications.)</p>
```
Comment (jpetto) [inline bedrock/security/templates/security/client-bug-bounty.html]: Looks like the word "be" is missing in this section: **...for a bug must rated a "sec-critical"...**

Comment (jpetto) [inline bedrock/security/templates/security/client-bug-bounty.html]: Can we make the URL to the wiki a functioning link?

File: bedrock/security/templates/security/client-bug-bounty.html [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+
+        <p>All bounties paid will be the discretion of the Mozilla Bounty Committee, which will evaluate the severity of reported issues with the help of engineers who work on the affected code. Security researchers are invited to participate in the assignment of ratings, but final decisions on the rating are at the discretion of the Bounty Committee.</p>
```
Comment (jpetto) [inline bedrock/security/templates/security/client-bug-bounty.html]: Should the beginning of this sentence read: "All bounties paid will be **at** the discretion of the..."?

File: bedrock/security/templates/security/client-bug-bounty.html [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+
+        <li>  Or, if an existing vulnerability can be demonstrated to be exploitable though additional research by the reporter, additional compensation can be earned for the same bug. Research might also uncover extremely severe, complex, or interesting problem areas that were previously unreported or unknown issues. Examples of severe or complex bugs would be: Use After Free bugs that also allow for ASLR bypass; bypassing the Firefox security wrappers to allow content to manipulate browser components, or a vulnerability that allows you to break out of a sandboxed process.</li>
+      </ul>
+
+      <p>We reserve the right not to pay bounties for security bugs in or caused by additional third party software (e.g. binary plugins, extensions) not bundled by Mozilla in a release.</p>
+
+    <table class="table">
+    <thead>
+      <tr>
+        <th>Novel vulnerability and explit, new form of exploitation or an exceptional vulnerability</th>
```
Comment (jpetto) [inline bedrock/security/templates/security/client-bug-bounty.html]: Typo - *explit*.

File: bedrock/security/templates/security/client-bug-bounty.html [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+        <th>High quality bug report with clearly exploitable critical vulnerability<sub>1</sub></th>
+        <th>High quality bug report of a critical or high vulnerability<sub>2</sub></th>
+        <th>Minimun for a high or critical vulnerability<sub>3</sub></th>
```
Comment (jpetto) [inline bedrock/security/templates/security/client-bug-bounty.html]: Typo - *Minimun*

File: bedrock/security/templates/security/client-bug-bounty.html [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+        <td>$3,000</td>
+        <td>$500 - $2500</td>
+      </tr>
+    </tbody>
+</table>
+<p><sup>1</sup>This is a report that includes details on exploitation of the vulnerability. A good example of this is a vulnerability that can gain remote code execution without the need of another vulnerability. </p>
+
+<p><sup>2</sup>This is a high quality report that includes minimized test cases and clear stack traces.</p>
+
+<p><sup>3</sup> this is a bug that includes a fuzzer report or a crash dump. </p>
```
Comment (jpetto) [inline bedrock/security/templates/security/client-bug-bounty.html]: Nit - capitalize "this" for consistency.

File: bedrock/security/templates/security/client-bug-bounty.html [hunk 8]
```diff
@@ -0,0 +1,101 @@
...
+
+      <h2>Claiming a Bug Bounty</h2>
+
+      <p>To claim a bounty:</p>
+      <ul>
+        <li>Make sure you have a Bugzilla account</li>
+          <li>File a bug at https://bugzilla.mozilla.org describing the security issue.</li>
```
Comment (jpetto) [inline bedrock/security/templates/security/client-bug-bounty.html]: Let's make this URL an active link.

File: bedrock/security/templates/security/client-bug-bounty.html [hunk 9]
```diff
@@ -0,0 +1,101 @@
...
+          <li>When creating the bug, be sure to check the box near the bottom of the entry form that marks this bug report as confidential.</li>
+          <li>Attach a "proof of concept" testcase, or point to explicit code that identifies the vulnerability or exploit.  While not required, such a testcase will help us judge submissions more quickly and accurately.</li>
+          <li>If you have debug output or output from a tool demonstrating the issue, please include it in the bug. (If it is very long, please attach it to the bug as an attached file.)</li>
+          <li>Notify the Mozilla Security Group by email to security@mozilla.org and include the number of the bug you filed and a brief summary of the issue.</li>
+    </ul>
+
+    <p>We ask that you be available to follow along and provide further information on the bug as needed, and invite you to work together with Mozilla engineers in reproducing, diagnosing, and fixing the bug. As part of this process we will provide you full access to participate in our internal discussions about the bug; for more information read our policy for handling security bugs.</p>
```
Comment (jpetto) [inline bedrock/security/templates/security/client-bug-bounty.html]: Is there a relevant URL we could link to for the "our policy for handling security bugs" text?

File: bedrock/security/templates/security/web-bug-bounty.html [hunk 10]
```diff
@@ -0,0 +1,50 @@
...
+        <li>Employees of the Mozilla Foundation and its subsidiaries are ineligible.</li>
+      </ul>
+      <p>If two or more people report the bug together, or working independently at approximately the same time, the reward will be divided between them.</p>
+
+      <h2>Reward Guidelines</h2>
+
+      <p>Mozilla will pay a bounty for website and service  security bugs as detailed below. All security bugs must follow the following general criteria to be eligible:</p>
+      <ul>
+        <li>Eligible security bugs may be present in any of eligible sites or services as outlined <a href="{{ url('security.bug-bounty.faq-webapp') }}">here</a></li>
+        <li>The security rating given by Bounty Committee for a bug must be "sec-critical", "sec-high" or "sec-moderate" in order for it to be eligible for a bounty. (See <a href="https://wiki.mozilla.org/WebAppSec/Web_App_Severity_Ratings">Web App Ratings</a> for details of the rating qualifications.)</li>
```
Comment (jpetto) [inline bedrock/security/templates/security/web-bug-bounty.html]: Should maybe be "...given by *the* Bounty Committee..."?

File: bedrock/security/templates/security/web-bug-bounty.html [hunk 11]
```diff
@@ -0,0 +1,50 @@
...
+        <li>All bounties paid will be the discretion of the Mozilla Bounty Committee, which will evaluate the severity of reported issues with the help of engineers who work on the affected code. Security researchers are invited to participate in the assignment of ratings, but final decisions on the rating are at the discretion of the Bounty Committee.</li>
```
Comment (jpetto) [inline bedrock/security/templates/security/web-bug-bounty.html]: Should be "..will be *at* the discretion of the..."?

File: bedrock/security/templates/security/web-bug-bounty.html [hunk 12]
```diff
@@ -0,0 +1,50 @@
...
+      </ul>
+    <h2>Rewards Amounts</h2>
+
+    <p>Vulnerabilities found in Mozilla Web applications and services are eligible for bounties between $500 (USD) and $4000 (USD), for extraordinary or critical vulnerabilities in valid web applications or Mozilla services.  Rewards for some critical sites can be even higher (e.g. FxAccounts, FindMyDevice, Update Services, Bugzilla)</p>
+    <p>Bounties will be awarded for sec-critical,  sec-high, or sec-moderate rated security bugs that meet the following criteria:</p>
+    <ul>
+      <li>Security bug is present in the web properties outlined in the Web Application Security Bounty FAQ.</li>
```
Comment (jpetto) [inline bedrock/security/templates/security/web-bug-bounty.html]: Can we link the text "Web Application Security Bounty FAQ" to a URL?

File: bedrock/security/templates/security/web-bug-bounty.html [hunk 13]
```diff
@@ -0,0 +1,50 @@
...
+
+      <li>Security bug is on the list of sites which part of the bounty. See the eligible bugs section of the Web Application Security Bounty FAQ for the list of sites which is included under the bounty.</li>
```
Comment (jpetto) [inline bedrock/security/templates/security/web-bug-bounty.html]: Looks like something is missing from this first sentence.

Comment (jpetto) [inline bedrock/security/templates/security/web-bug-bounty.html]: Again, would be nice to link the text "Web Application Security Bounty FAQ" somewhere (if possible).

File: bedrock/security/templates/security/web-bug-bounty.html [hunk 14]
```diff
@@ -0,0 +1,50 @@
...
+      <li>More information about this program can be found in the Web Application Security Bounty FAQ.</li>
```
Comment (jpetto) [inline bedrock/security/templates/security/web-bug-bounty.html]: And again here.

**Comment by jpetto** [comment]
A bit of cleanup needed, but overall looking good. Would be nice to consistently format/indent the HTML as well. (Us developers are an "orderly-to-a-fault". bunch :smile:)

After you've made a second commit addressing the review comments, I'll help you rebase with master (to deal with the `legal-docs` submodule thing) and squash your commits (to keep the git history tidy).

Thanks!

File: bedrock/security/templates/security/web-bug-bounty.html (hunk 10 above)
Comment (jpetto) [inline bedrock/security/templates/security/web-bug-bounty.html]: Still need to update this line?

File: bedrock/security/templates/security/bug-bounty/hall-of-fame.html [hunk 15]
```diff
@@ -43,7 +43,7 @@ <h3 data-accordion-role="tab">{{ _('2015') }}</h3>
             <li>Armin Razmdjou</li>
             <li>Atte Kettunen</li>
             <li>Enrique Rando</li>
-            <li>Hanno Bock</li>
+            <li>Hanno B&oumlck</li>
```
Comment (jpetto) [inline bedrock/security/templates/security/bug-bounty/hall-of-fame.html]: Should add a semicolon at the end of the HTML entity:

`Hanno B&ouml;ck`

**Pull request merged**